src/test/java/org/rutebanken/tiamat/repository/search/StopPlaceQueryFromSearchBuilderTest.java

```diff
@@ -0,0 +1,23 @@
+package org.rutebanken.tiamat.repository.search;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class StopPlaceQueryFromSearchBuilderTest {
+
+    @Test
+    public void handleCommonWordsInQuery() {
+
+        StopPlaceQueryFromSearchBuilder stopPlaceQueryFromSearchBuilder = new StopPlaceQueryFromSearchBuilder();
+        stopPlaceQueryFromSearchBuilder.commonWordsToIgnore.add("des");
```

Comment:
Maybe it's better to use constructor argument for common words to ignore? So that the class is more or less immutable?

Reply:
like this ?

```
/**
     * Configure some common words to be skipped during stop place search by name.
     */
    private Set<String> commonWordsToIgnore;

    public StopPlaceQueryFromSearchBuilder(@Value("${stopPlaces.search.commonWordsToIgnore}") Set<String> commonWordsToIgnore) {
        this.commonWordsToIgnore = commonWordsToIgnore;
    }
```

Reply:
Yes. Very nice. Does it handle empty set?